src/_P026_Sysinfo.ino

```diff
@@ -202,19 +213,23 @@ boolean Plugin_026(uint8_t function, struct EventStruct *event, String& string)
       // return decode(bytes,
       //  [header, uint24, uint24, int8, vcc, pct_8, uint8, uint8, uint8, uint8, uint24, uint16],
       //  ['header', 'uptime', 'freeheap', 'rssi', 'vcc', 'load', 'ip1', 'ip2', 'ip3', 'ip4', 'web', 'freestack']);
+      // on ESP32 you can add 'internaltemperature' of type int16 (1e2) to the list
       uint8_t index = 0;
-      string     += LoRa_addInt(P026_get_value(index++), PackedData_uint24);  // uptime
-      string     += LoRa_addInt(P026_get_value(index++), PackedData_uint24);  // freeheap
-      string     += LoRa_addFloat(P026_get_value(index++), PackedData_int8);  // rssi
-      string     += LoRa_addFloat(P026_get_value(index++), PackedData_vcc);   // vcc
-      string     += LoRa_addFloat(P026_get_value(index++), PackedData_pct_8); // load
-      string     += LoRa_addInt(P026_get_value(index++), PackedData_uint8);   // ip1
-      string     += LoRa_addInt(P026_get_value(index++), PackedData_uint8);   // ip2
-      string     += LoRa_addInt(P026_get_value(index++), PackedData_uint8);   // ip3
-      string     += LoRa_addInt(P026_get_value(index++), PackedData_uint8);   // ip4
-      string     += LoRa_addInt(P026_get_value(index++), PackedData_uint24);  // web
-      string     += LoRa_addInt(P026_get_value(index++), PackedData_uint16);  // freestack
-      event->Par1 = index;                                                    // valuecount
+      string += LoRa_addInt(P026_get_value(index++), PackedData_uint24);             // uptime
+      string += LoRa_addInt(P026_get_value(index++), PackedData_uint24);             // freeheap
+      string += LoRa_addFloat(P026_get_value(index++), PackedData_int8);             // rssi
+      string += LoRa_addFloat(P026_get_value(index++), PackedData_vcc);              // vcc
+      string += LoRa_addFloat(P026_get_value(index++), PackedData_pct_8);            // load
+      string += LoRa_addInt(P026_get_value(index++), PackedData_uint8);              // ip1
+      string += LoRa_addInt(P026_get_value(index++), PackedData_uint8);              // ip2
+      string += LoRa_addInt(P026_get_value(index++), PackedData_uint8);              // ip3
+      string += LoRa_addInt(P026_get_value(index++), PackedData_uint8);              // ip4
+      string += LoRa_addInt(P026_get_value(index++), PackedData_uint24);             // web
+      string += LoRa_addInt(P026_get_value(index++), PackedData_uint16);             // freestack
+      #  if FEATURE_INTERNAL_TEMPERATURE
+      string += LoRa_addInt(P026_get_value(index++) * 100.0f, PackedData_int16_1e2); // internal temperature in 0.01 degrees
```

Comment:
Please don't add this here (yet) as we also need to update the decoder JS code.
And since we're locked on the LoRaWAN decoder for TTN, I doubt we can add this here.
Problem is that the TTN decoder scripts are now limited to 5k. Only exception is for existing decoders. So not sure if we can update the decoder... ever.